lib/ui/views/schedule_view.dart

```diff
@@ -0,0 +1,212 @@
+      vsync: this,
+      duration: const Duration(milliseconds: 400),
+    );
+
+    _animationController.forward();
+  }
+
+  @override
+  void dispose() {
+    _animationController.dispose();
+    _calendarController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) =>
+      ViewModelBuilder<ScheduleViewModel>.reactive(
+          viewModelBuilder: () => ScheduleViewModel(),
+          onModelReady: (model) {
+            if (model.settings.isEmpty) {
+              model.loadSettings(_calendarController);
+            }
+          },
+          builder: (context, model, child) => BaseScaffold(
+                isLoading: model.busy(model.isLoadingEvents),
+                isInteractionLimitedWhileLoading: false,
+                appBar: AppBar(
+                  title: Text(AppIntl.of(context).title_schedule),
+                  centerTitle: false,
+                  automaticallyImplyLeading: false,
+                  actions: _buildActionButtons(model),
+                ),
+                body: Stack(children: [
+                  Column(
+                    children: [
+                      _buildTableCalendar(model),
+                      const SizedBox(height: 8.0),
+                      const Divider(indent: 8.0, endIndent: 8.0, thickness: 1),
+                      const SizedBox(height: 6.0),
+                      Center(
+                          child: Text(_dateFormat.format(model.selectedDate),
+                              style: Theme.of(context).textTheme.headline5)),
+                      const SizedBox(height: 2.0),
+                      Expanded(
+                          child: model.selectedDateEvents.isEmpty
+                              ? Center(
+                                  child: Text(
+                                      AppIntl.of(context).schedule_no_event))
+                              : _buildEventList(model.selectedDateEvents))
+                    ],
+                  ),
+                ]),
+              ));
+
+  /// Build the square with the number of [events] for the [date]
+  Widget _buildEventsMarker(DateTime date, List events) {
+    return Positioned(
+      right: 1,
+      bottom: 1,
+      child: AnimatedContainer(
+        duration: const Duration(milliseconds: 200),
+        decoration: BoxDecoration(
+          color: _calendarController.isSelected(date)
+              ? _selectedColor
+              : _defaultColor,
+        ),
+        width: 16.0,
+        height: 16.0,
+        child: Center(
+          child: Text(
+            '${events.length}',
+            style: const TextStyle().copyWith(
+              color: Colors.white,
+              fontSize: 12.0,
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  /// Build the calendar
+  Widget _buildTableCalendar(ScheduleViewModel model) => TableCalendar(
+        //TODO uncomment when https://github.com/aleksanderwozniak/table_calendar/issues/164 is close
```

Comment:
reminder @apomalyn

Reply:
That is a reminder, this TODO should stay there until the PR in tablecalendar is closed